<?xml version="1.0"?>
<!--[ Khelifa Baizid, Jul 2015 ]-->

<launch>
  <!--arg name="robot_name"/-->
  <node pkg="map_send_torob" type="map_send_torob" name="map_send_torob" output="screen">
    <!--param name="ip"          value="192.168.1.9"/-->
    <param name="ip"             value="10.1.16.73"/>
    <param name="map_topic"      value="/map"/>
    <param name="pose_topic"     value="/posegmapping"/>
    <param name="pose_is_needed" value="false"/>
    <param name="x_initial"      value="15.73"/>
    <param name="y_initial"      value="0.0"/>
    <param name="compress_map"   value="true"/>
    <param name="use_torob_map"  value="true"/>

  </node>

  <!--node pkg="rviz" type="rviz" name="rviz" output="screen"/-->   
</launch>

